includes/application/handleAppl.php

```diff
@@ -159,7 +159,26 @@
 		
 		$curDate = date('Y-m-d'); 
 		
-		if( ( strtotime($curDate) - strtotime($genDate) ) > 10*24*60*60 )
+		$sqlconn = new mysqli ( $eorgDBserver, $eorgDBuser, $eorgDBpasswd, $eorgDBname );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->